programmers/체육복.js

```diff
@@ -0,0 +1,14 @@
+function solution(n, lost, reserve) {
+    let answer = 0;
+    answer = (n - lost.length);
+    for (let i = 0; i < reserve.length; i++) {
+        if (reserve[i] === lost[i]) {
+            reserve.splice(reserve.indexOf(lost[i]), 1);
+            answer --;
+        }
```

Comment:
우선 reserve[i]와 lost[i]가 절대 같은 상황이 나올 수가 없지
lost는 잃어버린 학생이고 reserve는 여벌이 있는 학생인데 말이 안되는 코드임
```suggestion
        if (reserve[i] === lost[i]) {
            reserve.splice(reserve.indexOf(lost[i]), 1);
            answer --;
        }
```